src/lib/errorStatusCode.ts

```diff
@@ -0,0 +1,14 @@
+// please do not change the status code numbers
+// you can add new ones but not changing the existing ones
```

Comment:
@dennisseah I assume the guidance is to add increasing error status codes with spaces in between to give ourselves room to add related error statuses?

Reply:
precisely